package.json

```diff
@@ -16,14 +16,19 @@
     "resetdb": "yarn workspace wiki0 resetdb && yarn workspace todo0 resetdb",
     "migratedb": "yarn workspace wiki0 migratedb && yarn workspace todo0 migratedb",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx --cache",
-    "lint:fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix"
+    "lint:fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
+    "setup:env": "bash scripts/setup-env.sh",
```

Comment:
This is a good step 1, but in order to make it usable across all OS, should we need to either:
1. Require Windows users to WSL in the README
2. Rewrite scripts to use npm packages or js scripts? Instead of cmdline ops, would it be more accessible using `npx glob` or a js script with `fs.copyFile()`?

Thoughts?

Reply:
@alisaduncan  Just to clarify, with the first option, do you mean we’d need to ask Windows users to install WSL and then provide a separate set of instructions in the README for them? I’m not sure that’s the most straightforward path, since it adds extra steps and complexity for Windows users.

**With the second approach** –  (rewriting scripts in Node.js), it seems more sensible, since Node.js is already a requirement for the project, the JS scripts should work cross-platform out of the box. We can easily test this, and it keeps the setup simple for everyone. Personally, I think the second approach makes more sense for our use case.

Reply:
@iamspathan will reach out to discuss on Slack. As for yarn, we need to leave it for now unless there's a clear reason it's causing a barrier in env setup.